lib/logster/cache.rb

```diff
@@ -0,0 +1,16 @@
+module Logster
+  class Cache
+    def initialize(age = 2)
+      @age = age
+      @hash = { created_at: Time.now.to_f }
+    end
+
+    def get
+      if !@hash.key?(:data) || @hash[:created_at] + @age < Time.now.to_f
```

Comment:
I hate pointing at micro optimisations... but its worth noting ... `Process.clock_gettime(Process::CLOCK_MONOTONIC)` this is good for single process clock, its cheaper to ask for this cause you allocate one less object and is slightly more accurate.